torch/csrc/jit/script/class_type.cpp

```diff
@@ -112,8 +112,35 @@ size_t ClassType::addConstant(
   return slot;
 }
 
-c10::optional<IValue> ClassType::getConstant(const std::string& name) const {
-  TORCH_CHECK(constantNames_.size() == constantValues_.size());
+IValue ClassType::getConstant(const std::string& name) const {
+  TORCH_INTERNAL_ASSERT(constantNames_.size() == constantValues_.size());
```

Comment:
Same comment as the other one; let's re-use `findConstant` instead of duplicating the code.